### DISCUSSION TITLE
Terminal Session

### DISCUSSION THREAD

**Discussion by stephannn** [rustdesk/rustdesk-server-pro#91] [Ideas]
As an IT admin I use a lot PowerShell remote session to check and fix clients without calling the user.

Tools like Beyond Trust allow to create like an SSH tunnel to the device for exactly this purpose. Don't know if you have something like that on your agenda, but I think it would make the tool greater. Of course, the option has to be optional ;)

**Comment by rustdesk** [comment]
Some feature request there on RustDesk repo. In our plan, but not the highest priority.

**Comment by stephannn** [comment]
alrighty, I think this post covers it quite well: https://github.com/rustdesk/rustdesk/discussions/2840
